package.json

```diff
@@ -11,6 +11,7 @@
     "fs-plus": "^3.0.0",
     "fuzzaldrin": "^2.0",
     "fuzzaldrin-plus": "^0.6.0",
+    "fuzzy-native": "git://github.com/rafeca/fuzzy-native.git#electron-v2",
```

Comment:
Maybe you should publish this to NPM as `@atom/fuzzy-native` or something. That's what we've done for a few forks that we've had in the past, like `nsfw` and `react`.

Reply:
Sounds good, I'll do that